packages/atomic/src/components/formats/atomic-format-currency.tsx

```diff
@@ -0,0 +1,51 @@
+import {Component, Element, Prop, State, h} from '@stencil/core';
+import {dispatchNumberFormatEvent} from './format-common';
+
+/**
+ * The `atomic-format-currency` component is used for formatting currencies.
+ * It will set the numerical format on compatible parents according to the options.
```

Comment:
```suggestion
 * The numerical format of compatible parents will be set according to the currency property of this component.
```

Reply:
If my interpretation is wrong on this suggestion, disregard the following suggestions that are similar.